CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ NOTE: [`epaint`](crates/epaint/CHANGELOG.md), [`eframe`](crates/eframe/CHANGELOG
 * Plot widget - allow disabling zoom and drag for x and y separately [#2901](https://github.com/emilk/egui/pull/2901) (thanks [@OmegaJak](https://github.com/OmegaJak)!)
 * Add character limit to `TextEdit` [#2816](https://github.com/emilk/egui/pull/2816) (thanks [@wzid](https://github.com/wzid)!)
 * Add `egui::Modifiers::contains` [#2989](https://github.com/emilk/egui/pull/2989) (thanks [@Wumpf](https://github.com/Wumpf)!)
+* `std::borrow::Cow<'_, str>` now implements `TextBuffer` [#3164](https://github.com/emilk/pull/3164)
```

Comment:
This was in the wrong place. But nevermind adding it to the changelog - I now do that using `scripts/generate_changelog.py` just before releasing

```suggestion
```